helm/coder/values.yaml

```diff
@@ -111,6 +111,16 @@ coder:
     #     - update
     #     - watch
 
+    # coder.serviceAccount.workspaceNamespaces -- Grant this service account permissions 
```

Comment:
Above we have `workspacePerms`, `enableDeployments` and `extraRules`. I think these should be duplicated into `workspaceNamespaces` to support the case of having no additional permissions in the deployment namespace, but extra permissions in the workspace namespaces.